### PR TITLE
Feature/dependencies

### PR DESCRIPTION
**Pull request by floriantschopp**
As discussed, move dependencies to separate  submodules.

### CODE REVIEW

**Comment by floriantschopp** [comment]
Should we deprecate the maplab_dependencies then?

**Comment by smauq** [comment]
@floriantschopp seems to have just timed out at 2 hours and jenkins killed it. Maybe because nothing is ccached at the moment the first builds might be horribly long?

File: dependencies/LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
+Apache License
```
Comment (smauq) [inline dependencies/LICENSE]: Do we need this twice, since it's already in the root of the maplab folder?

File: dependencies/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+maplab_dependencies
```
Comment (smauq) [inline dependencies/README.md]: Also this is kind of an irrelevant file now?

**Pull request merged**